examples/cfd/darcy_physics_informed/darcy_physics_informed_fno.py

```diff
@@ -110,6 +107,14 @@ def main(cfg: DictConfig):
         padding=cfg.model.fno.padding,
     ).to(device)
 
+    phy_informer = PhysicsInformer(
+        required_outputs=["diffusion_u"],
+        equations=darcy,
+        grad_method="finite_difference",
+        device=device,
+        fd_dx=1 / 240,
```

Comment:
Is this a known constant? Maybe name it or add comment?

Reply:
Done :)